### PR TITLE
bowerize. fixes #39

### LINKED ISSUE #39: Bower
Would be nice to download this using bower

### PR DESCRIPTION
**Pull request by buzzdecafe**

### CODE REVIEW

**Comment by buzzdecafe** [comment]
if no one has any objections, i'll register this on bower

File: bower.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
+{
+  "name": "ramda-fantasy",
+  "description": "Fantasy Land compatible types for easy integration with Ramda",
+  "version": "0.1.0",
```
Comment (TheLudd) [inline bower.json]: Current version is `0.2.0`

**Comment by TheLudd** [comment]
Can this really be published and used? If anyone would like to download this they still need node to handle requiring files since we don't have a main build right?

File: bower.json (hunk 1 above)
Comment (buzzdecafe) [inline bower.json]: nice catch

**Comment by buzzdecafe** [comment]
correct, `"moduleType": [ "node" ]` 
I suppose we'll want to wrap in UMD or browserify, but idk if it's strictly necessary at `v0.2.0`

File: package.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
   "author": "Michael Hurley <mh@buzzdecafe.com> (buzzdecafe.com)",
   "name": "ramda-fantasy",
   "description": "Fantasy Land compatible types for easy integration with Ramda",
-  "version": "0.1.0",
+  "version": "0.2.0",
```
Comment (davidchambers) [inline package.json]: You should `git rebase master`. The master branch is already at `0.2.0`.

Comment (buzzdecafe) [inline package.json]: thanks david

File: package.json [hunk 3]
```diff
@@ -15,7 +15,7 @@
     "test": "mocha"
   },
   "dependencies": {
-    "ramda": "^0.13.0"
+    "ramda": "^0.14.0"
```
Comment (davidchambers) [inline package.json]: This upgrade required no changes?

Comment (buzzdecafe) [inline package.json]: `npm test` looks good

File: bower.json [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+  "moduleType": [
+    "node"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/ramda/ramda-fantasy.git"
+  },
+  "keywords": [
+    "functional", "abstract", "adt", "datatype", "type"
+  ],
```
Comment (davidchambers) [inline bower.json]: xyz would rewrite this as:

```json
  "keywords": [
    "functional",
    "abstract",
    "adt",
    "datatype",
    "type"
  ],
```

If you plan to merge #41 (or the alternative PR), you might like to reformat this list.

Comment (buzzdecafe) [inline bower.json]: ok will do. might as well alphabetize while i'm at it

**Comment by buzzdecafe** [comment]
we good here?

**Comment by davidchambers** [comment]
LGTM

**Pull request merged**